src/richText.js

```diff
@@ -145,8 +113,17 @@ define([
      * if the editor does not exist.
      *
      * @param input - editor jQuery HTML element.
-     * @param form - form object; used for creating bubbles.
-     * @param options - rich text bubble options.
+     * @param form - form object, functions used:
```

Comment:
This is now the form. One of the original goals for this was to totally remove refs of form from the widget so it was totally independent of vellum. I think it should be fairly trivial to do that now that it's fully documented though.